Dockerfile

```diff
@@ -56,10 +56,12 @@ RUN rm -fr /root/loxilb-io/loxilb/.git
 RUN rm -fr /root/loxilb-io/loxilb/.github
 RUN mkdir -p /root/loxilb-io/loxilb/
 RUN cp /usr/local/sbin/loxilb /root/loxilb-io/loxilb/loxilb
-#RUN /usr/local/sbin/mkllb_bpffs
 
-#RUN cd /root/loxilb-io/loxilb/ && make test
- 
+# Optional files, only apply when files exist
+COPY ./loxilb.rep* /root/loxilb-io/loxilb/loxilb
```

Comment:
This change is causing docker build CICD case to fail [here](https://github.com/loxilb-io/loxilb/actions/runs/4914153413/jobs/8775928927) . It could be due to lower docker version issue used in Ubuntu 20.04. But we need something generic which works across Ubuntu20.04+. Request to double-check.

Reply:
Hi @TrekkieCoder 
It seems like this feature has been supported for a while ([ref](https://medium.com/driven-by-code/flexible-docker-images-with-optional-copy-1e7ef84d44dd)).
Can we update the docker version that CICD used currently?